receiver/sqlserverreceiver/templates/sqlServerQuerySample.tmpl

```diff
@@ -0,0 +1,45 @@
+SELECT
```

Comment:
There is no limit of rows returned specified in this query. Should we consider limiting the number of sample queries we report in one scrape?

Reply:
added config to limit rows returned